src/Actions/ManagesContacts.php

```diff
@@ -83,6 +101,36 @@ public function findOrCreateContact($email, $firstName, $lastName, $orgid = null
         return $this->createContact($email, $firstName, $lastName, $orgid);
     }
 
+    /**
+     * Updates a contact.
+     *
+     * @param Contact|int|string $id
+     * @param string|null        $email
+     * @param string|null        $firstName
+     * @param string|null        $lastName
+     * @param null               $orgid
+     *
+     * @return Contact|null
+     */
+    public function updateContact($id, $email, $firstName, $lastName, $orgid = null)
+    {
+        $id = $this->getContactId($id);
```

Comment:
I try to avoiding too many flexible variables. This one can be normalised when you pass the Contact object into this function.

Reply:
Agreed. Will update.

Reply:
@stefanius Flexible variables?